### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by devongovett** [audiocogs/pitch.js#4]
What do you think of doing an annotated version of the source using [docco](https://github.com/jashkenas/docco)?  Obviously pitch.js isn't particularly hard to use, but I think a docco version of the source (if annotated well) could be interesting to people and help them understand the source, and maybe we'd get more outside contributors because of that.  Anyway, just a thought.

**Comment by jussi-kalliokoski** [comment]
Yes, it's worth thinking about. For now, I uploaded at least some docs generated by the build scripts I have in sink.js :D

**Comment by jussi-kalliokoski** [comment]
Oops, it's missing the important methods. Oh well, will fix later.

**Comment by coleww** [comment]
This is way old, but I just found this library and I'm not clear on how to use it. Some docs of the available methods would be very helpful, or especially a simple code example that gets the audio context, hooks up the mic, and displays pitch. 

I'm trying to make an app that converts guitar notes into brainfuck code and then runs it

**Comment by crackofdusk** [comment]
@coleww You can check out the example here: https://github.com/audiocogs/pitch.js-demo

**Comment by clu3** [comment]
@crackofdusk i've setup the site but i only see some wheels and some access-mic permission flash setting, what am i supposed to do with this demo? Thanks

**Comment by crackofdusk** [comment]
@clu3 Authorize mic access, then try singing a note. The pitch detector will use mic input as its input.

**Comment by clu3** [comment]
Thanks @crackofdusk I got it running.
I'm playing with pitch.js. I'm trying to compare the pitch between a user's recording and a given sample mp3 file, just to see if the English learner speaks English in the right tone. Hopefully pitch.js can really give me the data so I can draw 2 graphs that users can actually see and compare for themselves
